### PR TITLE
Feat: training modules dashboard (resolves #400)

### LINKED ISSUE #400: Training modules: learner dashboard
For users who have completed training courses or who signed up to complete training courses, their dashboard should show completed and in-progress courses as well as links to re-send email certificates for their completed courses.

### PR DESCRIPTION
**Pull request by chosww**
Add training modules dashboard. #400 

### Prerequisites

If this PR changes PHP code or dependencies:

- [x] I've run `composer format` and fixed any code formatting issues.
- [x] I've run `composer analyze` and addressed any static analysis issues.
- [x] I've run `php artisan test` and ensured that all tests pass.
- [x] I've run `composer localize` to update localization source files and committed any changes.

If this PR changes CSS or JavaScript code or dependencies:

- [x] I've run `npm run lint` and fixed any linting issues.
- [x] I've run `npm run build` and ensured that CSS and JavaScript assets can be compiled.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,7 +1,7 @@
 APP_NAME="The Accessibility Exchange"
 APP_ENV=development
 APP_KEY=
-APP_DEBUG=false
+APP_DEBUG=true
```
Comment (jobara) [inline .env.example]: Is this an intentional change?

File: app/Models/Choice.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Spatie\Translatable\HasTranslations;
+
+class Choice extends Model
```
Comment (jobara) [inline app/Models/Choice.php]: I think this was removed in your other training related changes and should be removed from here as well.

File: app/Models/Question.php [hunk 3]
```diff
@@ -33,4 +37,24 @@ public function quizzes(): BelongsToMany
     {
         return $this->belongsToMany(Quiz::class);
     }
+
+    public function getChoices()
```
Comment (jobara) [inline app/Models/Question.php]: All this choice related stuff here looks old as well.

File: database/factories/ChoiceFactory.php [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+namespace Database\Factories;
+
+use App\Models\Question;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Choice>
+ */
+class ChoiceFactory extends Factory
```
Comment (jobara) [inline database/factories/ChoiceFactory.php]: Another bit the should likely be removed.

File: database/migrations/2022_11_21_194859_add_minimum_choices_to_questions_table.php [hunk 5]
```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
```
Comment (jobara) [inline database/migrations/2022_11_21_194859_add_minimum_choices_to_questions_table.php]: is this also in need of removal?

**Review by jobara — CHANGES_REQUESTED** [review]
Looks like you need to remove all the old Choice related models and etc.

File: .env.example (hunk 1 above)
Comment (chosww) [reply .env.example]: Oh, this isn't intentional, I accidentally commit this file.

**Comment by chosww** [comment]
Thanks for your review @jobara, there are mistakes caused during merging dev branch and resolving some conflicts. I've been checking git diff to make sure, and I will update this PR based on your PR feedback.

**Review by jobara — APPROVED** [review]

**Pull request merged**